src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/help-forceSandbox.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    <p>Controls whether the "Use Groovy Sandbox" is shown in the system to users without Overall/Administer permission.</p>
+    <p>This can be used to simplify the UX in highly secured environments where all pipelines and any other groovy execution are
+        required to run in the sandbox (ie. running arbitrary code is never approved)</p>
```

Comment:
```suggestion
    <p>This can be used to simplify the UX in highly secured environments where all Pipelines and any other Groovy execution are
        required to run in the sandbox (i.e., running arbitrary code is never approved).</p>
```